### ISSUE TITLE
[bug] indent + brace-style 'allman': TypeScript enum opening brace is indented

### ISSUE THREAD

**Issue by PindaPixel** [eslint-stylistic/eslint-stylistic#52]
When using:
```json
"rules": {
    "@stylistic/brace-style": ["warn", "allman"],
    "@stylistic/indent": ["warn", indent_size],
}
```

I expect the following to be correct:
```ts
export enum MyEnum
{
    First,
    Second,
}
```

Instead, the following occurs:

```ts
export enum MyEnum 
{
// ^ Expected indentation of 4 spaces but found 0.
    First,
    Second,
}
```

**Comment by Eugeno** [comment]
Looks like @stylistic/indent for TS is broken, try to use @stylistic/js/indent
https://github.com/eslint-stylistic/eslint-stylistic/issues/46

**Comment by PindaPixel** [comment]
> Looks like @stylistic/indent for TS is broken, try to use @stylistic/js/indent #46

Thanks, that works better, but not perfect. It now doesn't have any opinion on the indentation of the brace. Good enough for now, though.

**Comment by PindaPixel** [comment]
https://github.com/eslint-stylistic/eslint-stylistic/pull/261
